app/controllers/reviews_controller.rb

```diff
@@ -6,10 +6,20 @@ def new
   def create
     @card = current_user.cards.find(review_params[:card_id])
 
-    if @card.verify_translation(review_params[:user_translation], review_params[:translation_time])
-      redirect_to :back, notice: t('views.reviews.flash_messages.user_translated_correct')
-    else
-      redirect_to :back, notice: t('views.reviews.flash_messages.user_translated_incorrect')
+    respond_to do |format|
+      if @result = @card.verify_translation(review_params[:user_translation], review_params[:translation_time])
+        format.html {
+          redirect_to :back,
+          notice: t('views.reviews.flash_messages.user_translated_correct')
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.